Add instrumentation (#910)
github/view_component: docs/guide/instrumentation.md

---
layout: default
title: Instrumentation
parent: Building ViewComponents
---

# Instrumentation

> You can enable ActiveSupport notifications for all ViewComponents with `instrumentation_enabled` option:

Reviewer: ```suggestion
To enable ActiveSupport notifications, use the `instrumentation_enabled` option:
```

```ruby
# config/application.rb
# Enable ActiveSupport notifications for all ViewComponents
config.view_component.instrumentation_enabled = true
```

> Now you can subscribe to `render.view_component` event:

Reviewer: ```suggestion
Then subscribe to the `render.view_component` event:
```
